lib/sync_checker/formats/publication_check.rb

```diff
@@ -57,6 +58,16 @@ def expected_details_hash(edition)
       def rendering_app
         Whitehall::RenderingApp::WHITEHALL_FRONTEND
       end
+
+      def filter_documents_for_locale(edition, locale)
```

Comment:
Is it worth extracting this? - basically the same as the method in https://github.com/alphagov/whitehall/pull/2982/files#diff-08abbfe2cb3a245251d671a5077347bb